helpdesk_mgmt/views/helpdesk_ticket_templates.xml

```diff
@@ -37,42 +44,55 @@
                 />
                 <t t-set="title">Tickets</t>
                 <t t-set="url" t-value="'/my/tickets'" />
+                <t t-set="text">Follow and comments your helpdesk tickets</t>
                 <t t-set="placeholder_count" t-value="'ticket_count'" />
+                <t t-set="action">
+                    <form method="POST" t-attf-action="/new/ticket">
+                        <input
+                            type="hidden"
+                            name="csrf_token"
+                            t-att-value="request.csrf_token()"
+                        />
+                        <button
+                            name="create_new_ticket"
+                            type="action"
+                            class="btn btn-sm btn-primary"
+                        >
+                            <i class="fa fa-plus" /> New
+                        </button>
+                    </form>
+                </t>
             </t>
-            <form method="POST" t-attf-action="/new/ticket">
-                <input
-                    type="hidden"
-                    name="csrf_token"
-                    t-att-value="request.csrf_token()"
-                />
-                <button
-                    name="create_new_ticket"
-                    type="action"
-                    class="btn btn-primary"
-                    style="float: right; margin-right: 0px; margin-bottom: 5px;"
-                >New Ticket</button>
-            </form>
         </div>
     </template>
+    <template id="portal_searchbar" inherit_id="portal.portal_searchbar">
+        <xpath expr="//div[@t-if='searchbar_sortings']" position="before">
+            <t t-if="action">
+                <t t-out="action" />
```

Comment:
Why not putting here directly `<form>` and so on instead of having to declare a variable?

Reply:
The purpose of this was to keep the initial logic where everything was set using `t-set`.

Reply:
Well, I think it's not needed and complicate the code.

Reply:
Ok, do I also need to edit the other one (`portal_docs_entry`) ?

Reply:
I think the goal of this refactoring is to embed everything of the button in one template, and then only do the `t-call` on each place where needed, so yes.

Reply:
I thought about moving the form in its own template and using a `t-call` to display it but there is a minor difference between both buttons (`btn-sm` class is added for the button in the "My account" display). Is there a better way to handle it ?

In fact, if I move the `<form>`, I would still need a variable to enable/disable its display, otherwise
![image](https://github.com/user-attachments/assets/528e19bd-2175-480d-9d98-fd4459c5a056)

Reply:
You can pass by variables some of that values, like the class or the enabling, but reduce it to the minimum. Anyway, I see weird that you need to do that instead of just inheriting from the specific button and that's it.